[docs][CI/CD Guide] Missing EXPO_TOKEN usage explained (#12401)
expo/expo: docs/pages/guides/setting-up-continuous-integration.md

$ npx expo login --non-interactive -u <EXPO USERNAME>
```

> Alternatively you can generate an access token under your profile settings and set an environment variable named `EXPO_TOKEN` with it. After that you can skip whole login command.

Reviewer: ```suggestion
Alternatively, you can [generate an access token under your Expo account settings](https://docs.expo.io/accounts/programmatic-access/) and configure it as an environment variable named `EXPO_TOKEN`. 
```

> Alternatively, you can [generate an access token under your Expo account settings](https://docs.expo.io/accounts/programmatic-access/) and configure it as an environment variable named `EXPO_TOKEN`. 

Reviewer: ```suggestion
Alternatively, you can [generate an access token under your Expo account settings](/accounts/programmatic-access.md) and configure it as an environment variable named `EXPO_TOKEN`. 
```
